docker/alpine/README.md

```diff
@@ -0,0 +1,45 @@
+# SFTPgo with Docker and Alpine
+
+This DockerFile is made to build image to host multiple instances of SFTPgo started with different users.
+
+The volume for the configuration is not mandatory, but it will be necessary to configure SFTPgo with environment variables.
+
+### Example
+> 1003 is a custom uid:gid for this instance of SFTPgo
+```
+# Prereq on docker host
+sudo groupadd -g 1003 sftpgrp && \
+  sudo useradd -u 1003 -g 1003 sftpuser -d /home/sftpuser/ && \
+  sudo -u sftpuser mkdir /home/sftpuser/{conf,data} && \
```

Comment:
Creating user and group is a clean way but not strictly required, you could at least document that 

```
chown 1003:1003 <folder>
```
works too.

Reply:
It's just an example for the comprehension. Yes, username and group name are not mandatory